.circleci/config.yml

```diff
@@ -108,79 +108,23 @@ jobs:
       - run: make enclave
       - run: ./tools/ci/sgx_test
   geth-postgres:
-    resource_class: xlarge
-    docker:
-      - image: smartcontract/builder:1.0.25
-        environment:
-          TYPEORM_NAME: ci
-          TYPEORM_HOST: explorerdb
-      - image: circleci/postgres:11
-        environment:
-          POSTGRES_USER: circleci_postgres
-          POSTGRES_DB: circleci_test
-          POSTGRES_INITDB_ARGS: "--lc-collate=C --lc-ctype=C"
-      - image: circleci/postgres:11
-        name: explorerdb
-        environment:
-          POSTGRES_USER: root
-          POSTGRES_DB: explorer_test
-          POSTGRES_INITDB_ARGS: "--lc-collate=C --lc-ctype=C"
-    environment:
-      DATABASE_URL: postgres://circleci_postgres@localhost:5432/circleci_test?sslmode=disable
+    resource_class: large
+    machine:
+      image: ubuntu-1604:201903-01
```

Comment:
Any reason for using Ubuntu?

Reply:
this was chosen from the circleci docs, which is also used in the forks test